Mathlib/Data/MvPolynomial/Supported.lean

```diff
@@ -60,15 +60,15 @@ noncomputable def supportedEquivMvPolynomial (s : Set σ) : supported R s ≃ₐ
     (AlgEquiv.ofInjective (rename ((↑) : s → σ)) (rename_injective _ Subtype.val_injective)).symm
 #align mv_polynomial.supported_equiv_mv_polynomial MvPolynomial.supportedEquivMvPolynomial
 
-@[simp]
+--@[simp]  -- porting note: the `simpNF` linter complained about this lemma.
```

Comment:
In what sense? Did it claim that simp can prove this? Or that the LHS doesn't simplify?

Reply:
```
-- Found 2 errors in 117592 declarations (plus 289330 automatically generated ones) in mathlib with 12 linters

/- The `simpNF` linter reports:
SOME SIMP LEMMAS ARE NOT IN SIMP-NORMAL FORM.
see note [simp-normal form] for tips how to debug this.
https://leanprover-community.github.io/mathlib_docs/notes.html#simp-normal%20form -/
-- Mathlib.Data.MvPolynomial.Supported
#check MvPolynomial.supportedEquivMvPolynomial_symm_C.{u_1, u} /- LINTER FAILED:
simplify fails on left-hand side:
failed to synthesize
  CommMonoidWithZero { x // x ∈ MvPolynomial.supported R s }
(deterministic) timeout at 'typeclass', maximum number of heartbeats (20000) has been reached (use 'set_option synthInstance.maxHeartbeats <num>' to set the limit) -/
#check MvPolynomial.supportedEquivMvPolynomial_symm_X.{u_1, u} /- LINTER FAILED:
simplify fails on left-hand side:
failed to synthesize
  CommMonoidWithZero { x // x ∈ MvPolynomial.supported R s }
(deterministic) timeout at 'typeclass', maximum number of heartbeats (20000) has been reached (use 'set_option synthInstance.maxHeartbeats <num>' to set the limit) -/
```

Reply:
Thanks, Eric!  I reinstated the `simp` attribute and unlinted it.